### ISSUE TITLE
[Task]: Audit and Fix Accessibility Issues with API Docs

### ISSUE THREAD

**Issue by acouch** [HHS/simpler-grants-gov#1258]
### Summary

The API docs will be public after the API launch. The UI should be checked for accessibility. The spec we should be testing against is 508.

A testing tool needs to be identified.

Lighthouse in the chromium-based browsers is an option.

This task is to:

- [x] Choose tool and run initial audit
- [x] Fix issues

<details>
<summary>Lighthouse results</summary>
Two issues were identified with Lighthouse:

![image](https://github.com/HHS/simpler-grants-gov/assets/512243/eee34509-f766-4819-b254-c70667deaa3f)

</details>

### Acceptance criteria

- [x] API docs pass designated tool
